Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,6038 @@
+        raise ValueError(ErrorMessage.VIRTUAL_SERVER.value)
+
+    http_service = args.get('http_service')
+    https_service = args.get('https_service')
+    if not (http_service or https_service):
+        raise ValueError(ErrorMessage.PROTOCOL.value)
+    if args.get('deployment_mode') and args['deployment_mode'] not in [
+            'HTTP Content Routing', 'Single Server/Server Balance'
+    ]:
+        raise ValueError(ErrorMessage.DEPLOYMENT_MODE.value)
+    if args['deployment_mode'] == 'Single Server/Server Balance' and not args.get('server_pool'):
+        raise ValueError(ErrorMessage.SERVER_POOL.value)
+    if version == ClientV2.API_VER:
+        scripting = args.get('scripting')
+        if args.get('scripting') and args['scripting'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.SCRIPTING.value)
+        scripting_list = args.get('scripting_list')
+        if scripting == 'enable' and not scripting_list:
+            raise ValueError(ErrorMessage.SCRIPTING_LIST.value)
+        if args.get('certificate_type') and args['certificate_type'] not in [
+                'Local', 'Multi Certificate', 'Letsencrypt'
+        ]:
+            raise ValueError(ErrorMessage.CERTIFICATE_TYPE.value)
+        if args.get('client_real_ip') and args['client_real_ip'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.CLIENT_REAL_IP.value)
+        if args.get('mach_once') and args['mach_once'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.MACH_ONCE.value)
+        if args.get('monitor_mode') and args['monitor_mode'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.MONITOR_MODE.value)
+        if args.get('redirect_to_https') and args['redirect_to_https'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.REDIRECT_2_HTTPS.value)
+        if args.get('retry_on') and args['retry_on'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.RETRY_ON.value)
+        if args.get('retry_on_http_layer') and args['retry_on_http_layer'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.RETRY_ON_HTTP_LAYER.value)
+        if args.get('retry_on_connect_failure') and args['retry_on_connect_failure'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.RETRY_ON_CONNECT_FAILURE.value)
+        if args.get('syn_cookie') and args['syn_cookie'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.SYN_COOKIE.value)
+        if args.get('url_case_sensitivity') and args['url_case_sensitivity'] not in ['enable', 'disable']:
+            raise ValueError(ErrorMessage.URL_CASE_SENSITIVITY.value)
+        half_open_thresh = arg_to_number(args.get('half_open_thresh'))
+        if half_open_thresh and not 10 <= half_open_thresh <= 10000:
+            raise ValueError(ErrorMessage.HALF_OPEN_THRESH.value)
+        arg_to_number(args.get('retry_on_cache_size'))
```

Comment:
is this here just to fail if it's not present? arg_to_number returns `None` if failing to parse so this line may be redundant.